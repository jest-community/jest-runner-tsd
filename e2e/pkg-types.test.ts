import { expect, test } from '@jest/globals';
import runJest from './runJest';

test('reads `types` property in package.json', async () => {
  expect(await runJest('pkg-types')).toMatchSnapshot();
});
