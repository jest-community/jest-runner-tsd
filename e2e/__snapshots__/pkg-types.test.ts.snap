// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`reads \`types\` property in package.json 1`] = `
"PASS e2e/__fixtures__/pkg-types/types.test.ts
  âœ“
Test Suites: 1 passed, 1 total
Tests:       2 passed, 2 total
Snapshots:   0 total
Time:
Ran all test suites.
"
`;
